overlay_new.py

```diff
@@ -0,0 +1,198 @@
+from abc import ABC, abstractmethod
+from textwrap import wrap
+from time import time
+from typing import Callable, Tuple
+
+from canvas import Canvas, Colour
+from data import Data
+from overlay import Overlay
+
+class Drawable(ABC):
+    @abstractmethod
+    def draw_base(self, canvas: Canvas):
```

Comment:
Leave a comment on what the difference between `draw_base` and `draw_data` is. Eg. One is rendered once and the other is rendered upon every data point.